src/Pages/Overview/DisplayProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import './App.css'
+const enums = require('../../Enums.js')
+
+let toggle = false
+let toggleDelete = false
+let userToDelete = null
+let userToEdit = null
+
+// displaying children of member-board
+export function displayBoard (user, index) {
+  return (
+    <tr key={index}>
```

Comment:
You can remove the `key` element here. This component is not direclty inside of an `array.map` so it is ok. See [this article](https://reactjs.org/docs/lists-and-keys.html#keys) for more details

Reply:
Console give a key-warning when remove. Having it there solve the warning.